lib/class-wp-json-posts-controller.php

```diff
@@ -933,4 +940,206 @@ protected function prepare_links( $post ) {
+			'title',
+			'editor',
+			'author',
+			'excerpt',
+			'thumbnail',
+			'comments',
+			'revisions',
+			'page-attributes',
+			'post-formats',
+			);
+		$fixed_post_attributes = array(
+			'title',
+			'editor',
+			'author',
+			'excerpt',
+			'thumbnail',
+			'comments',
+			'revisions',
+			'post-formats',
+			);
+		$fixed_page_attributes = array(
+			'title',
+			'editor',
+			'author',
+			'excerpt',
+			'thumbnail',
+			'comments',
+			'revisions',
+			'page-attributes',
+			);
+		foreach( $post_type_attributes as $attribute ) {
+			if ( 'post' === $this->post_type && ! in_array( $attribute, $fixed_post_attributes ) )  {
+				continue;
+			} else if ( 'page' === $this->post_type && ! in_array( $attribute, $fixed_page_attributes ) ) {
+				continue;
+			} else if ( ! in_array( $this->post_type, array( 'post', 'page' ) ) && ! post_type_supports( $this->post_type, $attribute ) ) {
+				continue;
+			}
+
+			switch( $attribute ) {
+
+				case 'title':
+					$schema['properties']['title'] = array(
+						'description'     => 'The title for the object.',
+						'type'            => 'object',
+						'properties'      => array(
+							'raw'         => array(
+								'description'     => 'Title for the object, as it exists in the database.',
+								'type'            => 'string',
+								),
+							'rendered'    => array(
+								'description'     => 'Title for the object, transformed for display.',
+								'type'            => 'string',
+								),
+							),
+						);
+					break;
+
+				case 'editor':
+					$schema['properties']['content'] = array(
+						'description'     => 'The content for the object.',
+						'type'            => 'object',
+						'properties'      => array(
+							'raw'         => array(
+								'description'     => 'Content for the object, as it exists in the database.',
+								'type'            => 'string',
+								),
+							'rendered'    => array(
+								'description'     => 'Content for the object, transformed for display.',
+								'type'            => 'string',
+								),
+							),
+						);
+					break;
+
+				case 'author':
+					$schema['properties']['author'] = array(
+						'description'     => 'The ID for the author of the object.',
+						'type'            => 'integer',
+						);
+
+				case 'excerpt':
+					$schema['properties']['excerpt'] = array(
+						'description'     => 'The excerpt for the object.',
+						'type'            => 'object',
+						'properties'      => array(
+							'raw'         => array(
+								'description'     => 'Excerpt for the object, as it exists in the database.',
+								'type'            => 'string',
+								),
+							'rendered'    => array(
+								'description'     => 'Excerpt for the object, transformed for display.',
+								'type'            => 'string',
+								),
+							),
+						);
+					break;
+
+				case 'thumbnail':
+					$schema['properties']['featured_image'] = array(
+						'description'     => 'A featured image for the object.',
```

Comment:
This description is not very helpful.  Should be something like "ID of the featured image for the object."